### PR TITLE
Created new pattern service-pattern

### LINKED ISSUE #116: Block Patterns - Services
**Description**

🛠️ The goal of this issue is to track the implementation of the "Services" pattern.
🎨 [Figma](https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?type=design&node-id=565%3A10402&mode=design&t=LHSPHKUrPq556l59-1)

Services

### PR DESCRIPTION
**Pull request by gnanasekaranl**

### CODE REVIEW

File: patterns/services.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:image {"id":598,"sizeSlug":"full","linkDestination":"media"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/tt4_service.png" alt="" class="wp-image-598"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:heading {"level":3} -->
+<h3>Guiding your business through the project</h3>
```
Comment (luminuu) [inline patterns/services.php:18]: Please make all strings translateable, see other patterns how to do it.

File: patterns/services.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Experience the fusion of imagination and expertise with Études—the catalyst for architectural transformations that enrich the world around us.</p>
```
Comment (luminuu) [inline patterns/services.php:22]: See above

File: patterns/services.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button -->
+<div class="wp-block-button"><a class="wp-block-button__link wp-element-button" href="#">Learn More</a></div>
```
Comment (luminuu) [inline patterns/services.php:27]: See above

**Review by luminuu — CHANGES_REQUESTED** [review]

**Comment by MaggieCabrera** [comment]
Closing this since we already merged https://github.com/WordPress/twentytwentyfour/pull/94. Thanks for your contribution!

**Pull request closed without merging**